com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -664,6 +641,47 @@ public static ProbeSubdivisionResult BakeBricks(ProbeSubdivisionContext ctx)
             return result;
         }
 
+        // Converts brick information into positional data at kBrickProbeCountPerDim * kBrickProbeCountPerDim * kBrickProbeCountPerDim resolution
+        internal static void ConvertBricksToPositions(List<Brick> bricks, Vector3[] outProbePositions)
+        {
+            Matrix4x4 m = ProbeReferenceVolume.instance.GetRefSpaceToWS();
```

Comment:
Do we still need that?

Reply:
Not really for now, but because we need sooner rather than later moving the origin too I'd rather keep that there for now so the transition is easier (same reason why the transform is a bit everywhere)